packages/strapi-plugin-content-manager/services/utils/configuration/metadatas.js

```diff
@@ -10,29 +10,32 @@ const {
 const { formatContentTypeSchema } = require('../../ContentTypes');
 
 function createDefaultMetadatas(schema) {
-  return {
-    ...Object.keys(schema.attributes).reduce((acc, name) => {
-      acc[name] = createDefaultMetadata(schema, name);
-      return acc;
-    }, {}),
-    id: {
-      edit: {},
-      list: {
-        label: 'Id',
-        searchable: true,
-        sortable: true,
-      },
-    },
+  const idAttribute = _.get(schema, ['options', 'idAttribute'], 'id');
```

Comment:
same you should create a function in the attributes utils and export it to be used in here too

Reply:
Done